pkg/sysinfo/sysinfo.go

```diff
@@ -16,6 +16,9 @@ type SysInfo struct {
 	cgroupCpusetInfo
 	cgroupPids
 
+	// Whether the kernel supports cgroup namespaces or not
+	CgroupNamespaces bool
```

Comment:
Perhaps name this `...Enabled` (if it's `opt-in`) or `..Disabled` (if we'll make it `opt-out`) (see my other comment) to be consistent with other options

```suggestion
	CgroupNamespacesEnabled bool
```

Should this also be in a separate type, and embedded (as we did for the other features?)


edit: hm.. looks like we're not really consistent on the naming, so not sure

Reply:
I copied what was done for AppArmor and Seccomp - those were maybe poor examples to follow. :) I'm happy to break it out into an embedded type, if that's the preferred way of doing it.